pkgs/applications/editors/kakoune.cr/default.nix

```
@@ -0,0 +1,42 @@
+{ bat, crystal, fd, fzf, fetchFromGitHub, fetchurl, jq, lib, stdenv }:
+let
+  icon = fetchurl {
+    url = "https://github.com/mawww/kakoune/raw/master/doc/kakoune_logo.svg";
+    sha256 = "1x9gdrnrqgzvr1ixr6s8ff6cvz01yp2xdh5ad6nbh6p2d094h617";
+    name = "kakoune.svg";
+  };
+
+in crystal.buildCrystalPackage {
```

```suggestion
in crystal.buildCrystalPackage rec {
```
This will let us reference `${version}`.